internal/action/recipients.go

```diff
@@ -53,7 +53,7 @@ func (s *Action) recipientsList(ctx context.Context) []string {
 		return nil
 	}
 
-	return tree.List(0)
+	return tree.List(-1)
```

Comment:
Should we have a constant for this special value?
`tree.INF` or something?

Reply:
Sure, it involves importing tree in many places, but I guess it should be fine.